### PR TITLE
CMS: Templates. Update links in CMS education guide

### PR DESCRIPTION
**Pull request by katilp**
Addresses #1998
@tiborsimko does the Higgs example already have a record number?

### CODE REVIEW

File: cernopendata/modules/fixtures/data/docs/cms-guide-for-education/cms-guide-for-education.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Two examples of these specially prepared files are:
 
 You can also download these files directly from the records on this portal.
 
-To learn how to use the [CMS event display](/visualise/events/cms), click on `Need help?` link on the top right side of the screen. For suggestions for class room activities see [instructions for use of CMS Open Data in spreadsheets for schools and education](/record/FIXME).
+To learn how to use the [CMS event display](/visualise/events/cms), click on `Need help?` link on the top right side of the screen. For suggestions for class room activities see [Tutorials and worksheets for CMS Event Display](/record/5103).
```
Comment (RaoOfPhysics) [inline cernopendata/modules/fixtures/data/docs/cms-guide-for-education/cms-guide-for-education.md]: How'd I not catch this earlier? It should be "classroom activities".

Comment (katilp) [reply cernopendata/modules/fixtures/data/docs/cms-guide-for-education/cms-guide-for-education.md]: Thanks! I'll end up learning that one day 😉

**Comment by tiborsimko** [comment]
> does the Higgs example already have a record number?

Yes, 5500 is reserved.

**Comment by katilp** [comment]
OK, thanks! Added that and done other changes.

**Pull request merged**